recipes/protobuf/all/test_package/CMakeLists.txt

```diff
@@ -6,24 +6,19 @@ conan_basic_setup(TARGETS)
 
 find_package(Protobuf CONFIG REQUIRED)
 
-if (protobuf_LITE)
-    find_program(PROTOC_PROGRAM protoc)
-    if (NOT PROTOC_PROGRAM)
-       message(WARNING "Using protobuf-lite but protoc was not found")
-    endif()
-endif()
-
 add_executable(${PROJECT_NAME} test_package.cpp addressbook.proto)
 set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
 target_include_directories(${PROJECT_NAME} PRIVATE "${CMAKE_BINARY_DIR}")
 if (protobuf_LITE)
     target_link_libraries(${PROJECT_NAME} protobuf::libprotobuf-lite)
 else()
     target_link_libraries(${PROJECT_NAME} protobuf::libprotobuf)
-endif()
 
-protobuf_generate_cpp(PROTO_SRCS PROTO_HDRS TARGET ${PROJECT_NAME})
-protobuf_generate(LANGUAGE cpp TARGET ${PROJECT_NAME} PROTOS addressbook.proto)
+    find_program(PROTOC_PROGRAM protoc)
+    if (NOT PROTOC_PROGRAM)
+       message(WARNING "Protoc was not found")
+    endif()
 
-# # message(STATUS "PROTO_SRCS: ${PROTO_SRCS}")
-# # message(STATUS "PROTO_HDRS: ${PROTO_HDRS}")
+    protobuf_generate_cpp(PROTO_SRCS PROTO_HDRS TARGET ${PROJECT_NAME})
```

Comment:
But if you restrict `protobuf_generate_cpp` to non-lite then `addressbook.proto` wont be treated in lite case.
I’m missing something?

Reply:
You are correct. @uilianries did removed protoc from `lite=True` packages in previous comments so I adapted the test_package. We talked in Slack and we might just ending up packing everything for `lite=True`

https://github.com/conan-io/conan-center-index/pull/2037/commits/fa07c0e83954429ba4f2915496aa7633892bec7b

Reply:
Nice, thanks! The simpler the better.
Hope we can finish this PR. I’m interested on having grpc on CCI :-P